monitorrent/tests/plugins/clients/test_qbittorrent.py

```diff
@@ -0,0 +1,173 @@
+from monitorrent.plugins.clients.qbittorrent import QBittorrentClientPlugin
+from monitorrent.tests import DbTestCase, use_vcr
+from mock import patch
+
+
+@ddt
+class QBittorrentPluginTest(DbTestCase):
+    real_host = "http://192.168.100.5"
+    real_port = 8080
+    real_login = "admin"
+    real_password = "password"
+
+    bad_host = "http://fake.com"
+    bad_port = 1234
+    bad_login = "fake"
+    bad_password = "more_fake"
+
+    def test_settings(self):
+        plugin = QBittorrentClientPlugin()
+        settings = {'host': 'localhost', 'username': 'monitorrent', 'password': 'monitorrent'}
+        self.assertIsNone(plugin.get_settings())
+        plugin.set_settings(settings)
+        readed_settings = plugin.get_settings()
+
+        self.assertEqual({'host': 'localhost', 'port': None, 'username': 'monitorrent'}, readed_settings)
+
+    @use_vcr
+    def test_check_connection_successfull(self):
+        plugin = QBittorrentClientPlugin()
+        settings = {'host': self.real_host, 'port': self.real_port, 'username': self.real_login,
+                    'password': self.real_password}
+        plugin.set_settings(settings)
+        self.assertTrue(plugin.check_connection())
+
+    @patch('requests.Session.post')
+    def test_check_connection_failed(self, post_mock):
+        response = Response()
+        response.status_code = 404
+        post_mock.return_value = response
+
+        plugin = QBittorrentClientPlugin()
+        self.password_ = {'host': self.bad_host, 'port': self.bad_port, 'username': self.bad_login,
+                          'password': self.bad_password}
+        settings = self.password_
+        plugin.set_settings(settings)
+        self.assertFalse(plugin.check_connection())
+
+    @use_vcr
+    @patch("requests.Session.get")
+    def test_find_torrent(self, get_mock):
+        response = Response()
+        response._content = b'[{"added_on":"2016-04-09T20:24:17","completion_on":null,"dlspeed":0,"eta":8640000,"f_l_piece_prio":false,"force_start":false,"hash":"d27c8e30f0c12afe84735b57f6463266b118acea","label":"","name":"Ment.v.zakone.9.2015.HDTVRip.Files-x","num_complete":12,"num_incomplete":42,"num_leechs":0,"num_seeds":0,"priority":1,"progress":0.073527365922927856,"ratio":0.022764846784523337,"save_path":"Downloads","seq_dl":false,"size":5776162816,"state":"pausedDL","super_seeding":false,"upspeed":0}]'
+        get_mock.return_value = response
+
+        plugin = QBittorrentClientPlugin()
+        torrent_hash = "D27C8E30F0C12AFE84735B57F6463266B118ACEA"
+        settings = {'host': self.real_host, 'username': self.real_login,
+                    'password': self.real_password}
+        plugin.set_settings(settings)
+        torrent = plugin.find_torrent(torrent_hash)
+        self.assertEqual(torrent['date_added'], datetime(2016, 4, 9, 20, 24, 17, tzinfo=UTC).replace(tzinfo=reference.LocalTimezone()).astimezone(utc))
+        self.assertEqual(torrent['name'], 'Ment.v.zakone.9.2015.HDTVRip.Files-x')
```

Comment:
should it be tzinfo=UTC replaced with tzinfo=pytz.reference.LocalTimezone()
then replace can be omitted